src/components/NotifiNotifications/Announcements.jsx

```diff
@@ -0,0 +1,165 @@
+    <Timeline className={classes.sectionBody}>
+      {timelineCards.map((card, i) => {
+        const isNotFirst = i !== 0;
+        const isNotLast = i !== timelineCards.length - 1;
+        const isActive = card.isActive;
+        const nextIsActive = isNotLast && timelineCards[i + 1].isActive;
+        return (
+          <TimelineItem key={i}>
+            <TimelineOppositeContent
+              className={classes.verticallyCenterContent}
+              align="right"
+              variant="body2"
+              color="textSecondary"
+            ></TimelineOppositeContent>
+            <TimelineSeparator>
+              {isNotFirst && (
+                <TimelineConnector
+                  className={
+                    isActive
+                      ? clsx(classes.connector, "active")
+                      : classes.connector
+                  }
+                />
+              )}
+              {isNotLast && (
+                <TimelineConnector
+                  className={
+                    nextIsActive
+                      ? clsx(classes.connector, "active")
+                      : classes.connector
+                  }
+                />
+              )}
+            </TimelineSeparator>
+            <TimelineContent>
+              <Paper
+                elevation={0}
+                className={
+                  isActive ? clsx(classes.paper, "active") : classes.paper
+                }
+              >
+                <Box className={classes.titleSection}>
+                <Typography
+                  variant="h6"
+                  component="h3"
+                  className={
+                    isActive
+                      ? clsx(classes.cardTitle, "active")
+                      : classes.cardTitle
+                  }
+                >
+                  {
+                    <span role="img" aria-label={card.emojiAria}>
+                      {card.emoji}
+                    </span>
+                  }{" "}
+                  {card.title}
+                </Typography>
+                <Typography
+                  variant="h6"
+                  component="h3"
+                  className={classes.date}
+                >
+                  {formatDateTime(card.date)}
+                </Typography>
+                </Box>
+                {!!card.body && (
+                  <Typography
+                    className={
+                      isActive
+                        ? clsx(classes.cardBody, "active")
+                        : classes.cardBody
+                    }
+                  >
+                    {card.body}
+                  </Typography>
+                )}
+              </Paper>
+            </TimelineContent>
+          </TimelineItem>
+        );
+      })}
+    </Timeline>
+    </>
+    )
+}
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Usually `styles` goes to top of the page.

Reply:
Thank you for the note! Thought adding it at the end would help with readability, didn't know that should have it at the top! Will update on my end :)